lib/Event.php

```diff
@@ -19,6 +19,96 @@
+    const INVOICE_SENT                         = 'invoice.sent';
+    const INVOICE_UPCOMING                     = 'invoice.upcoming';
+    const INVOICE_UPDATED                      = 'invoice.updated';
+    const INVOICEITEM_CREATED                  = 'invoiceitem.created';
+    const INVOICEITEM_DELETED                  = 'invoiceitem.deleted';
+    const INVOICEITEM_UPDATED                  = 'invoiceitem.updated';
+    const ORDER_CREATED                        = 'order.created';
+    const ORDER_PAYMENT_FAILED                 = 'order.payment_failed';
+    const ORDER_PAYMENT_SUCCEEDED              = 'order.payment_succeeded';
+    const ORDER_UPDATED                        = 'order.updated';
+    const ORDER_RETURN_CREATED                 = 'order_return.created';
+    const PAYOUT_CANCELED                      = 'payout.canceled';
+    const PAYOUT_CREATED                       = 'payout.created';
+    const PAYOUT_FAILED                        = 'payout.failed';
+    const PAYOUT_PAID                          = 'payout.paid';
+    const PAYOUT_UPDATED                       = 'payout.updated';
+    const PLAN_CREATED                         = 'plan.created';
+    const PLAN_DELETED                         = 'plan.deleted';
+    const PLAN_UPDATED                         = 'plan.updated';
+    const PRODUCT_CREATED                      = 'product.created';
+    const PRODUCT_DELETED                      = 'product.deleted';
+    const PRODUCT_UPDATED                      = 'product.updated';
+    const RECIPIENT_CREATED                    = 'recipient.created';
+    const RECIPIENT_DELETED                    = 'recipient.deleted';
+    const RECIPIENT_UPDATED                    = 'recipient.updated';
+    const REVIEW_CLOSED                        = 'review.closed';
+    const REVIEW_OPENED                        = 'review.opened';
+    const SIGMA_SCHEDULED_QUERY_RUN_CREATED    = 'sigma.scheduled_query_run.created';
+    const SKU_CREATED                          = 'sku.created';
+    const SKU_DELETED                          = 'sku.deleted';
+    const SKU_UPDATED                          = 'sku.updated';
+    const SOURCE_CANCELED                      = 'source.canceled';
+    const SOURCE_CHARGEABLE                    = 'source.chargeable';
+    const SOURCE_FAILED                        = 'source.failed';
+    const SOURCE_MANDATE_NOTIFICATION          = 'source.mandate_notification';
+    const SOURCE_TRANSACTION_CREATED           = 'source.transaction.created';
+    const TOPUP_CREATED                        = 'topup.created';
+    const TOPUP_FAILED                         = 'topup.failed';
+    const TOPUP_SUCCEEDED                      = 'topup.succeeded';
+    const TRANSFER_CREATED                     = 'transfer.created';
+    const TRANSFER_REVERSED                    = 'transfer.reversed';
+    const TRANSFER_UPDATED                     = 'transfer.updated';
+    const PING                                 = 'ping';
```

Comment:
Maybe just keep this one alphabetized with the others.

Reply:
It follows the exact order of the events on https://stripe.com/docs/api#event_types
But I can move it up if you want?

Reply:
Yeah, move it. We should probably change it in the API reference too.

Reply:
I think the reason it's at the bottom is because it's the only one that has no object attached. In other words; it's not really an event you can process in any way.

Reply:
Yeah. I just think that in this case it'd be better to keep alphabetical order because that sort of information isn't really present. It's generally expected that this sort of list of constants is arranged.